apps/src/netsim/NetSimLobby.js

```diff
@@ -215,86 +273,149 @@ NetSimLobby.prototype.refreshShardList_ = function () {
   });
 };
 
+NetSimLobby.prototype.useRandomShard = function () {
+  this.randomShardID = 'netsim_' + utils.createUuid();
+  this.useShard(this.randomShardID);
+};
+
+NetSimLobby.prototype.useShard = function (shardID) {
+  this.shardSelector_.empty();
+
+  $('<option>')
+      .val(shardID)
+      .html('My Private Network')
+      .appendTo(this.shardSelector_);
+
+  this.shardLinks_.
+      attr('href', this.buildShareLink(shardID)).
+      show();
```

Comment:
Can't necessarily find anything in our style guide that points at exactly what this should look like. I would have thought only 2 spaces of indent. Also, I would try to be consistent about whether the . is at the end of each line or the beginning.